plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_launcher_util.py

```diff
@@ -64,7 +64,9 @@ def launch_job_on_ray(
     singleton_state: Any,
 ) -> Any:
     if ray_remote:
-        run_job_ray = ray.remote(**ray_remote)(_run_job)
+        run_job_ray = ray.remote(**OmegaConf.to_container(ray_remote, resolve=True))(
```

Comment:
```suggestion
        run_job_ray = ray.remote(**OmegaConf.to_container(ray_remote, resolve=True, throw_on_missing=True))(
```
Using `throw_on_missing=True` would be consistent with the previous behavior.

Particularly, if `ray_remote` is a simple DictConfig with a missing value, e.g. if
```python
ray_remote = OmegaConf.create({"a": 123, "b": "???"})
```
then the previous behavior would be to throw a `MissingMandatoryValue` error when the `**` operator is invoked as `**ray_remote`.